APIDOC.md

```
@@ -10,17 +10,30 @@ Package smartmontools provides Go bindings for interfacing with smartmontools to
 
 The library wraps the smartctl command\-line utility and provides a clean, idiomatic Go API for accessing SMART information from storage devices.
 
+### Go 1.26 Optimizations
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)